core/dbt/events/proto_types.py

```diff
@@ -2159,7 +2159,6 @@ class TrackingInitializeFailure(betterproto.Message):
     exc_info: str = betterproto.string_field(2)
 
 
-@dataclass
 class EventBufferFull(betterproto.Message):
     """Z045"""
```

Comment:
This is a generated file. You need to remove the EventBufferFulll message from types.proto and run the protoc compiler (as described in the events README.md).

Reply:
Thanks! Will take care of this!